scripts/production/run_healthchecks.pl

```diff
@@ -0,0 +1,210 @@
+
+=head1 SYNOPSIS
+
+  perl run_healthchecks.pl $COMPARA_REG compara_curr -g ComparaAll
+
+  perl run_healthchecks.pl \
+    --reg_conf registry_configuration_file --reg_alias compara_curr \
+    -t org.ensembl.healthcheck.testcase.compara.MLSSTagGERPMSA --repair
+
+=head1 ARGUMENTS
+
+The script reads these arguments and passes the other ones straight to run-configurable-testrunner.sh
+
+=head2 DATABASE SETUP
+
+=over
+
+=item B<[--url mysql://user[:passwd]@host[:port]/dbname]>
+
+URL of the database to test
+
+=item B<[--reg_conf registry_configuration_file]>
+
+The Bio::EnsEMBL::Registry configuration file. If none given and no URL is
+given, the one set in ENSEMBL_REGISTRY will be used if defined, if not
+~/.ensembl_init will be used.
+
+=item B<[--reg_type reg_type]>
+
+The "type" or "group" under which the database is to be found in the Registry.
+
+=item B<[--reg_alias|--reg_name name]>
+
+The name or "species" under which the database is to be found in the Registry.
+
+=item B<[--compara_master|--master_db name_or_url]>
+
+A reference to the master database to compare against. Can be either a
+Registry alias (assumed to be in the "compara" group/type) or a URL.
+Defaults to 'compara_master', i.e. a Registry lookup.
+
+=back
+
+=head2 HEALTHCHECK SETUP
+
+=over
+
+=item B<[--ensj-testrunner /path/to/run-configurable-testrunner.sh]>
+
+The path to run-configurable-testrunner.sh. If not given a default will be
+formed using the ENSEMBL_CVS_ROOT_DIR environment variable.
+
+=item B<[--ensj-json-config /path/to/ensj-healthcheck.json]>
+
+The path to ensj-healthcheck.json. If not given a default will be formed
+using the ENSEMBL_CVS_ROOT_DIR environment variable.
+
+=item B<[--repair]>
+
+Use this flag if you want to use the "repair" mode of the healtcheck.
+
+=back
+
+=cut
+
+use strict;
+use warnings;
+
+use File::Basename qw(dirname);
+use Getopt::Long qw(:config pass_through);
+use JSON qw(decode_json);
+
+use Bio::EnsEMBL::ApiVersion;
+use Bio::EnsEMBL::Registry;
+use Bio::EnsEMBL::Utils::IO qw(:slurp);
+
+use Bio::EnsEMBL::Compara::DBSQL::DBAdaptor;
+use Bio::EnsEMBL::Compara::Utils::Registry;
+
+my ($url, $reg_conf, $reg_type, $reg_alias, $master_db);
+my ($repair, $ensj_testrunner, $ensj_json_config);
+
+# Default values
+$master_db = 'compara_master';
+
+# Arguments parsing
+GetOptions(
+    'url=s'                         => \$url,
+    'reg_conf|regfile|reg_file=s'   => \$reg_conf,
+    'reg_type=s'                    => \$reg_type,
+    'reg_alias|regname|reg_name=s'  => \$reg_alias,
+    'master_db|compara_master=s'    => \$master_db,
+    'ensj-testrunner=s'             => \$ensj_testrunner,
+    'ensj-json-config=s'            => \$ensj_json_config,
+    'repair'                        => \$repair,
+);
+
+unless ($url or ($reg_conf and $reg_alias)) {
+    print "\nNeither --url nor --reg_conf and --reg_alias are defined. The URL should be something like mysql://ensro\@compara1:3306/kb3_ensembl_compara_59\nEXIT 2\n\n";
```

Comment:
Out of curiosity, why the `EXIT 2` bit?

Reply:
Unnecessary copy-paste. I have restored this to `exit 1`

Reply:
Sorry, I meant in the `print` command. Although I assume if you have changed below, in the printed string it should say `EXIT 1` too?
```suggestion
    print "\nNeither --url nor --reg_conf and --reg_alias are defined. The URL should be something like mysql://ensro\@compara1:3306/kb3_ensembl_compara_59\nEXIT 1\n\n";
```

Reply:
Sorry I didn't see it, the line was too long :D !
I'll remove the `EXIT`, we usually don't print it

```suggestion
   print "\nNeither --url nor --reg_conf and --reg_alias are defined. The URL should be something like mysql://ensro\@compara1:3306/kb3_ensembl_compara_59\n\n";
```